### DISCUSSION TITLE
NPE LabeLayer mWorker

### DISCUSSION THREAD

**Discussion by jvoyatz** [mapsforge/vtm#919] [General]
Hello,

What is this issue? 
`Attempt to invoke virtual method 'void org.oscim.layers.tile.vector.labeling.LabelLayer$Worker.submit(long)' on a null object reference`

How it can be resolved?

**Comment by devemux86** [comment]
There should be a complete log to understand a report.

You can debug your implementation and compare with the samples that work correctly.
